### DISCUSSION TITLE
MayExitFARW

### DISCUSSION THREAD

**Discussion by shmuz** [elfmz/far2l#1217] [Q&A] [answered]
При наличии `MayExitFARW`, выводящей диалог подтверждения, и попытке закрыть окно Фара крестиком, диалог не выводится, а Фар зависает. Это можно как-то исправить?

**Comment by elfmz** [comment, marked as answer]
скорее всего можно,

**Comment by elfmz** [comment]
пробуйте

**Comment by shmuz** [comment]
Работает как надо, спасибо.
